source/elements/oneDAL/source/notations.rst

```diff
@@ -4,4 +4,57 @@
 Mathematical Notations
 ======================
 
+.. list-table::
+   :widths: 27 73
+   :header-rows: 1
+
+   * - Notation
+     - Definition
+
+   * - :math:`N`
+     - Number of feature vectors in a :term:`dataset <Dataset>`.
+
+   * - :math:`p`
+     - Number of features in a :term:`dataset <Dataset>`.
+
+   * - :math:`x_i` or :math:`x'_i`
+     - The :math:`i`-th :term:`feature vector <Feature
+       vector>` of a :term:`training set <Training set>` or a :term:`inference
+       set <Inference set>` respectively.
+
+   * - :math:`y_i`
+     - The :math:`i`-th :term:`response <Response>` of
+       a :term:`training set <Training set>` given a priori.
+
+   * - :math:`y'_i`
+     - The :math:`i`-th :term:`response <Response>` that needs
+       to predicted by an inference algorithm given the feature vector
+       :math:`x'_i` from :term:`inference set <Inference set>`.
+
+   * - :math:`a \times b`
+     - Denotes that the corresponding matrix (dataset) has :math:`a` rows
+       (observations) and :math:`b` columns (features).
+
+   * - :math:`|A|`
+     - If :math:`A` is a set, this denotes cardinality of a set, i.e., the
+       number of elements in the set :math:`A`. If :math:`A` is a real number,
+       this denotes an absolute value of :math:`A`.
+
+   * - :math:`\|x\|_l`
+     - The :math:`L`-norm for a vector :math:`x \in \mathbb{R}^n` and a real
+       number :math:`l`.
```

Comment:
Is l a real, at least for the lib's needs?
would make sense to give few 'typical' values of l
minor: somewhere earlier we introduced x_i notation for feature vector and here we use the same for coordinate of the vector. The context should help differentiate the idea behind the use of x_i; we need to make sure the context is clear in the description of the algorithms or features